charts/prefect-server/README.md

```diff
@@ -152,9 +168,57 @@ the HorizontalPodAutoscaler.
+| backgroundServices.readinessProbe.config.failureThreshold | int | `3` | The number of consecutive failures allowed before considering the probe as failed. |
+| backgroundServices.readinessProbe.config.initialDelaySeconds | int | `10` | The number of seconds to wait before starting the first probe. |
+| backgroundServices.readinessProbe.config.periodSeconds | int | `10` | The number of seconds to wait between consecutive probes. |
+| backgroundServices.readinessProbe.config.successThreshold | int | `1` | The minimum consecutive successes required to consider the probe successful. |
+| backgroundServices.readinessProbe.config.timeoutSeconds | int | `5` | The number of seconds to wait for a probe response before considering it as failed. |
+| backgroundServices.readinessProbe.enabled | bool | `false` |  |
+| backgroundServices.resources.limits | object | `{"cpu":"1","memory":"1Gi"}` | the requested limits for the background-services container |
+| backgroundServices.resources.requests | object | `{"cpu":"500m","memory":"512Mi"}` | the requested resources for the background-services container |
+| backgroundServices.revisionHistoryLimit | int | `10` | the number of old ReplicaSets to retain to allow rollback |
+| backgroundServices.runAsSeparateDeployment | bool | `false` | Run background services (like scheduling) in a separate deployment. |
+| backgroundServices.serviceAccount.annotations | object | `{}` | additional service account annotations (evaluated as a template) |
+| backgroundServices.serviceAccount.create | bool | `true` | specifies whether a service account should be created |
+| backgroundServices.serviceAccount.name | string | `""` | the name of the service account to use. if not set and create is true, a name is generated using the common.names.fullname template |
+| backgroundServices.tolerations | list | `[]` | tolerations for background-services pod assignment |
 | commonAnnotations | object | `{}` | annotations to add to all deployed objects |
 | commonLabels | object | `{}` | labels to add to all deployed objects |
 | fullnameOverride | string | `"prefect-server"` | fully override common.names.fullname |
+| global.prefect.env | list | `[]` | array with environment variables to add to all deployments |
+| global.prefect.image.prefectTag | string | `"3-latest"` | prefect image tag (immutable tags are recommended) |
+| global.prefect.image.pullPolicy | string | `"IfNotPresent"` | prefect image pull policy |
+| global.prefect.image.pullSecrets | list | `[]` | prefect image pull secrets |
+| global.prefect.image.repository | string | `"prefecthq/prefect"` | prefect image repository |
+| global.prefect.prefectApiHost | string | `"0.0.0.0"` | sets PREFECT_SERVER_API_HOST |
+| global.prefect.prefectApiUrl | string | `"http://localhost:4200/api"` | sets PREFECT_API_URL |
```

Comment:
Per Eddie's comment in https://github.com/PrefectHQ/prefect-helm/pull/426#discussion_r1917622915, do we definitely need `global` here? that's usually for making values accessible to downstream/child charts

Reply:
interesting, is it _reserved_ for that purpose? If so, i can modify the top level key. I added the `global.prefect` key to prevent any of the sub values from accidentally being used in downstream charts.